### ISSUE TITLE
Integrate all ditto tools - Android

### ISSUE THREAD

**Issue by okdistribute** [getditto/demoapp-chat#127]
Let's add all tools to the demo app for chat on both iOS and Android

This includes the bluetooth permissions health tool, and anything else new that @bplattenburg 's team has been working on lately.

This was discussed in a call with Simon because he wants to be able to use the apps to check if the customer's device supports bluetooth, and requested some kind of tool to do that. Integrating into the existing demo apps is useful because it's easier to keep it up to date, installed, and pull it up quickly during a customer call.

**Comment by bplattenburg** [comment]
Depends on https://github.com/getditto/DittoSwiftTools/issues/70 and https://github.com/getditto/DittoAndroidTools/issues/24

**Comment by okdistribute** [comment]
This was also requested to see if the customer's device supports wifi aware

**Comment by bplattenburg** [comment]
@okdistribute Does anything in our existing tools show WiFiAware compatibility?

I'm wondering if that should to be its own story to take the existing WifiAwareChecker app and put that into DittoAndroidTools instead?

**Comment by bplattenburg** [comment]
After some discussion with @rajramsaroop we broke that out into https://github.com/getditto/DittoAndroidTools/issues/112

**Comment by rajramsaroop** [comment]
Update on this: Wifi Aware Checker has been updated and is now published to the play store: https://play.google.com/store/apps/details?id=live.ditto.wifiawarechecker

Adding Wifi Aware checker as a tool will be worked on next.

For integrating all tools into Ditto Chat I have associated this issue with a PR that adds the Tools Viewer to Ditto Chat for Android. Someone will need to create the iOS equivalent.
